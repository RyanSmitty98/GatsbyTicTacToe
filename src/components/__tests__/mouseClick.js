import React from "react";
import {cleanup, fireEvent, render} from '@testing-library/react';
import renderer from "react-test-renderer";
import {shallow} from 'enzyme';
import CheckboxWithLabel from '../CheckboxWithLabel';

describe('Test Button component', () => {
    it('Test click event', () => {
      // when someone clicks in the first box it will test to see if an x is placed there
      fireEvent.click(getByLabelText(/off/i));


    });
  });