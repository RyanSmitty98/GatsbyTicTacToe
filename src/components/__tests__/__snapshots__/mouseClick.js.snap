// Jest Snapshot v1, https://goo.gl/fbAQLP

exports


    <div>
        <button>
            X
        </button>
    <div>